### DISCUSSION TITLE
Event or Closure after reordering table?

### DISCUSSION THREAD

**Discussion by RX-77IT** [filamentphp/filament#6619] [Help] [answered]
Perchance, is it possible to set up an event after reordering a table with ->reorderable()? Or maybe pass a Closure that in some way is executed after reordering?

**Comment by danharrin** [comment]
I don't think its a feature at the moment, so please PR it

**Reply by RX-77IT** [reply]
I tried to write something, but failed miserably.  :-(

As a self taught php amateur, what should I know or study before attempting another try? Which file is the best to start modifying and experimenting on? 

The reason for why I need a closure comes from a change in the query for reordering records: that query no longer triggers the eloquent model updating event, which I'm using to know when to clean laravel's cache.

**Comment by RX-77IT** [comment, marked as answer]
I am such a newbie, <facepalm> it was way simpler than I thought...

In the ListRecords.php, just override the `reorderTable()` method

```php
public function reorderTable(array $order): void
{
    parent::reorderTable($order);

    // call events and do stuff
}
```

Credit to [danharrin](https://github.com/filamentphp/filament/pull/6852#issuecomment-1617656802) for helping us out!
